apps/vessel-history/features/map/map.hooks.ts

```diff
@@ -53,3 +64,93 @@ export interface Viewport extends LatLon {
   longitude: number
   zoom: number
 }
+
```

Comment:
I dont like to move this function to the shared libs because it has configuration of how leah want to open the link, I am not sure if another app need to use something similar how I will parametrize this. after split the function I will see if any of the functions make sense to move it